src/watertap_contrib/reflo/property_models/fo_draw_solution_properties.py

```diff
@@ -0,0 +1,812 @@
+#################################################################################
+# WaterTAP Copyright (c) 2020-2023, The Regents of the University of California,
+# through Lawrence Berkeley National Laboratory, Oak Ridge National Laboratory,
+# National Renewable Energy Laboratory, and National Energy Technology
+# Laboratory (subject to receipt of any required approvals from the U.S. Dept.
+# of Energy). All rights reserved.
+#
+# Please see the files COPYRIGHT.md and LICENSE.md for full copyright and license
+# information, respectively. These files are also available online at the URL
+# "https://github.com/watertap-org/watertap/"
+#################################################################################
+
+"""
+This module contains a basic property package for simple water treatment models.
+Volumetric flow and component concentration are used to determine mass flow. 
+"""
+from idaes.core import (
+    EnergyBalanceType,
+    MaterialBalanceType,
+    MaterialFlowBasis,
+    PhysicalParameterBlock,
+    StateBlock,
+    StateBlockData,
+    declare_process_block_class,
+)
+from idaes.core.base.components import Solvent, Solute
+from idaes.core.base.phases import LiquidPhase
+from idaes.core.solvers.get_solver import get_solver
+from idaes.core.util.misc import add_object_reference
+from idaes.core.util.initialization import (
+    fix_state_vars,
+    revert_state_vars,
+    solve_indexed_blocks,
+)
+from idaes.core.util.model_statistics import (
+    degrees_of_freedom,
+    number_unfixed_variables,
+)
+import idaes.logger as idaeslog
+import idaes.core.util.scaling as iscale
+
+from idaes.core.util.exceptions import (
+    ConfigurationError,
+    InitializationError,
+    PropertyPackageError,
+)
+
+from pyomo.environ import (
```

Comment:
import order should be

other pkg imports (e.g., pandas, numpy)
pyomo imports
idaes imports
watertap imports
watertap-reflo imports

Reply:
Order adjusted.